lib/show.rb

```diff
@@ -0,0 +1,14 @@
+class Show
+  attr_reader :name, :creator, :characters, :total_salary
+
+  def initialize(name, creator, characters)
+    @name = name
+    @creator = creator
+    @characters = characters
+  end
+
+  def total_salary
+    total_salary = @characters[0].salary + @characters[1].salary
+  end
```

Comment:
this total_salary method would work, but only if Hollywood ever made TV shows with exactly two characters... how would we test this for a show with only 1 actor? or 50?